flepimop/gempyor_pkg/src/gempyor/file_paths.py

```diff
@@ -198,12 +183,57 @@ def create_dir_name(
     """
     return os.path.dirname(
         create_file_name_without_extension(
-            run_id,
-            prefix,
-            1,
-            ftype,
-            inference_filepath_suffix,
-            inference_filename_prefix,
-            create_directory=False,
+            run_id, prefix, 1, ftype, inference_filepath_suffix, inference_filename_prefix, create_directory=False,
         )
     )
+
+
+def create_file_name_for_push(
+    flepi_run_index: str, prefix: str, flepi_slot_index: str, flepi_block_index: str
+) -> List[str]:
+    """
+    Generate a list of file names for different types of inference results.
+    
+    This function generates a list of file names based on the provided run index, prefix, slot index, 
+    and block index. Each file name corresponds to a different type of inference result, such as 
+    "seir", "hosp", "llik", etc. The file names are generated using the `create_file_name` function, 
+    with specific extensions based on the type: "csv" for "seed" and "parquet" for all other types.
+
+    Parameters:
```

Comment:
I think other functions in this file have been documented following the Google style guide, this looks like the numpy style guide maybe? We should pick a consistent choice, I have a preference for the Google style guide for brevity.